Library/Formula/when.rb

```diff
@@ -1,12 +1,21 @@
-require 'formula'
-
 class When < Formula
-  homepage 'http://www.lightandmatter.com/when/when.html'
-  url 'http://ftp.de.debian.org/debian/pool/main/w/when/when_1.1.33.orig.tar.gz'
-  sha1 'e314a64d74f79be07da1ade6d39a1fc51205f81d'
+  homepage "http://www.lightandmatter.com/when/when.html"
+  url "http://ftp.de.debian.org/debian/pool/main/w/when/when_1.1.34.orig.tar.gz"
```

Comment:
It'd be nice to use ` https://mirrors.kernel.org/debian/pool/main/ ` here if possible - The SSL/TLS protection is a nice bonus.